dbld/Makefile.am

```diff
@@ -6,4 +6,42 @@ EXTRA_DIST += \
 	dbld/generate-rpm-specfile \
 	dbld/generate-debian-directory \
 	dbld/shell \
-	dbld/rules
+	dbld/rules \
+	dbld/bootstrap	\
+	dbld/images/artful.dockerfile \
+	dbld/images/centos6.dockerfile \
+	dbld/images/centos7.dockerfile \
+	dbld/images/devshell-apt \
+	dbld/images/devshell-apt/all.txt \
+	dbld/images/devshell-apt/artful.txt \
+	dbld/images/devshell.dockerfile \
+	dbld/images/devshell-pip \
+	dbld/images/devshell-pip/all.txt \
+	dbld/images/helpers \
+	dbld/images/helpers/entrypoint.sh \
+	dbld/images/helpers/fake-sudo.sh \
+	dbld/images/helpers/functions.sh \
+	dbld/images/helpers/gosu.pubkey \
+	dbld/images/jessie.dockerfile \
+	dbld/images/README.md \
+	dbld/images/required-apt \
```

Comment:
List's off packages can be extended, by following the "distro-*.txt" pattern.
I recommend to include the whole directory in case of the "required" folders.

Reply:
Thank you! I will force push the update soon.